src/js/admin/screens/groupsScreen.js

```diff
@@ -0,0 +1,150 @@
+const tableHeader = `
+  <div class="tr groups-table-header">
+    <div class="td sr-img">Image</div>
+    <div class="td sr-number">Number</div>
```

Comment:
number by melo urcit identifikator skupiny,
key prejmenovat na order (poradi)

Reply:
jo měl jsem to naopak, takže z number je order a z key je pořadí